src/components/WorkOrders/CloseWorkOrderFormComponents/CloseWorkOrderFormReasonForClosing.tsx

```diff
@@ -38,25 +37,33 @@ const CloseWorkOrderFormReasonForClosing = (props: Props) => {
     }
   }, [reasonWatchedValue])
 
+  const [visitCompleted, noAccess] = CLOSURE_STATUS_OPTIONS
+
   return (
     <Radios
       labelSize="s"
       label="Reason for closing"
       name="reason"
-      options={CLOSURE_STATUS_OPTIONS.map((r) => ({
-        ...r,
-        defaultChecked: r.value === reason,
-        children: canRaiseAFollowOn ? (
-          r.value === 'Work Order Completed' ? (
+      options={[
+        {
+          text: visitCompleted.text,
+          value: visitCompleted.value,
+          defaultChecked: visitCompleted.value === reason,
+          children: (
             <FurtherWorkRadio
               error={errors?.followOnStatus}
+              visible={canRaiseAFollowOn && showFurtherWorkRadio}
               register={register}
-              visible={showFurtherWorkRadio}
               followOnStatus={followOnStatus}
             />
-          ) : null
-        ) : null,
-      }))}
+          ),
+        },
+        {
+          text: noAccess.text,
+          value: noAccess.value,
+          defaultChecked: noAccess.value === reason,
+        },
+      ]}
```

Comment:
Had quite a few bugs with this with the react-hook-form that were a fair bit easier to troubleshoot without the `.map`